DEVDOCS-3406b: GraphQL-related scope creep (#1496)
bigcommerce/dev-docs: docs/stencil-docs/javascript-and-event-hooks/adding-javascript.md

The npm package manager facilitates the management of third-party JavaScript components by placing each JavaScript component in the correct directories. However, as a developer, you will still need to edit your theme files to wire up the JavaScript component to expose it on your storefront. You will find several examples of this on the following pages.

## Page types and JavaScript API

> Stencil themes include an API for running JavaScript on a per-page basis. To properly write JavaScript for your theme, you will have the following page types available to you:

Reviewer: ```suggestion
Stencil themes include an API for running JavaScript on a per-page basis. To properly write JavaScript for your theme, you have the following page types available:
```

Author: ```suggestion
To help you encapsulate your theme's logic, Stencil themes include an API for running JavaScript on a per-page basis. The following page types are available:
```

@markcmurphy thoughts?
